stdlib/public/Windows/WinSDK.swift

```diff
@@ -222,21 +282,24 @@ public struct WindowsBool : ExpressibleByBooleanLiteral {
   }
 }
 
-extension WindowsBool : CustomReflectable {
+#if SWIFT_ENABLE_REFLECTION
+extension WindowsBool: CustomReflectable {
   /// Returns a mirror that reflects `self`.
   public var customMirror: Mirror {
     return Mirror(reflecting: boolValue)
   }
 }
+#endif
 
-extension WindowsBool : CustomStringConvertible {
+@_unavailableInEmbedded
```

Comment:
Really?

Reply:
Just keeping up with `DarwinBoolean`. I don't think that `WinSDK` is particularly useful with "embedded" yet. In theory, we could support IoT with the Windows IoT, but that is largely untested.